examples/debian_snapshot/BUILD.bazel

```diff
@@ -65,17 +66,36 @@ PACKAGES = [
     "@bullseye//perl",
 ]
 
+FLAT_REPO_PACKAGES = [
+    "@bullseye//r-base-core",
+]
+
+# Avoids `max depth issue` described here: https://github.com/GoogleContainerTools/rules_distroless/issues/36
+pkg_tar(
+    name = "flat_repo_packages_layer",
+    deps = select({
+        "@platforms//cpu:arm64": [
+            "%s/arm64" % package
+            for package in FLAT_REPO_PACKAGES
+        ],
+        "@platforms//cpu:x86_64": [
+            "%s/amd64" % package
+            for package in FLAT_REPO_PACKAGES
+        ],
+    }),
+)
+
```

Comment:
Found a way to easily avoid having to flatten stuff here in the test and depend on `rules_pkg`, etc.

You can simply use the `:data` target of the package so that you avoid pulling in all of the dependencies :)  Sure, that's "broken" for a real-life image but it'll work to show that the specific package is installed and thus it can be used to validate the test without having to bring in more dependencies and potentially have slower tests, etc.